index.html

```diff
@@ -21,6 +21,7 @@
           <a class="header-footer-nav-link extra-hover-underline" href="https://github.com/eiffel-community/eiffel">Protocol</a>
           <a class="header-footer-nav-link extra-hover-underline" href="https://eiffel-community.github.io/eiffel-sepia">Implementation Architecture</a>
           <a class="header-footer-nav-link extra-hover-underline" href="/community.html">Community</a>
+          <a class="header-footer-nav-link extra-hover-underline" href="/summit.html">Eiffel Summit</a>
```

Comment:
I guess this line should be added to other html files as well. For example community.html. Right?

Reply:
Done.